src/infrastructure/jira/jira-service.ts

```diff
@@ -46,25 +50,32 @@ class JiraService {
 			connectInstallation,
 		);
 
-		if (response.rejectedEntities.length) {
-			const { key, errors } = response.rejectedEntities[0];
-			throw JiraServiceSubmitDesignError.designRejected(key.designId, errors);
-		}
+		this.throwIfSubmitDesignResponseHasErrors(response);
+	};
 
-		// TODO: Confirm whether we need to consider the use case below as a failure and throw or just leave a warning.
-		if (response.unknownIssueKeys?.length) {
-			throw JiraServiceSubmitDesignError.unknownIssueKeys(
-				response.unknownIssueKeys,
-			);
-		}
+	updateDesigns = async (
+		designs: AtlassianDesign[],
+		connectInstallation: ConnectInstallation,
+	): Promise<void> => {
+		const response = await jiraClient.submitDesigns(
+			{
+				designs: designs.map((design) => ({
+					...design,
+					addAssociations: [],
+					removeAssociations: [],
```

Comment:
/nit just fyi, we've designed the api in such a way that if there's no intention changing any associations. Can just leave both `addAssociations` and `removeAssociations` as null

Reply:
Good to know! I'll switch the `[]`s to `null` and update our request body type to reflect this.